src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -110,66 +118,102 @@ const WorkflowDashboard = _.flow(
+              div({ style: styles.sectionTableLabel }, ['Start:']), div([start ? Utils.makeCompleteDate(start) : 'N/A']),
+              div({ style: styles.sectionTableLabel }, ['End:']), div([end ? Utils.makeCompleteDate(end) : 'N/A'])
+            ])
+          ]),
+          makeSection('Links', [
+            div({ style: { display: 'flex', flexFlow: 'row wrap', marginTop: '0.5rem' } }, [
+              h(Link, {
+                ...Utils.newTabLinkProps,
+                href: `${getConfig().jobManagerUrlRoot}/${workflowId}`,
+                style: { display: 'flex' },
+                tooltip: 'Job Manager'
+              }, [icon('tasks', { size: 18 }), ' Job Manager']),
+              h(Link, {
+                ...Utils.newTabLinkProps,
+                href: bucketBrowserUrl(`${bucketName}/${submissionId}/${workflowName}/${workflowId}`),
+                style: { display: 'flex', marginLeft: '1rem' },
+                tooltip: 'Execution directory'
+              }, [icon('folder-open', { size: 18 }), ' Execution Directory']),
+              h(Link, {
+                onClick: () => setShowLog(true),
+                style: { display: 'flex', marginLeft: '1rem' }
+              }, [icon('fileAlt', { size: 18 }), ' View execution log'])
+            ])
+          ])
+        ]),
+        failures && h(Collapse,
+          {
+            style: { marginBottom: '1rem' },
+            initialOpenState: true,
+            title: div({ style: Style.elements.sectionHeader }, [
+              'Workflow-Level Failures',
+              h(ClipboardButton, {
+                text: JSON.stringify(failures, null, 2),
+                style: { marginLeft: '0.5rem' },
+                onClick: e => e.stopPropagation() // this stops the collapse when copying
+              })
+            ])
+          }, [h(ReactJson, {
+            style: { whiteSpace: 'pre-wrap' },
+            name: false,
+            collapsed: 4,
+            enableClipboard: false,
+            displayDataTypes: false,
+            displayObjectSize: false,
+            src: restructureFailures(failures)
+          })]
+        ),
+        h(Collapse,
+          {
+            title: div({ style: Style.elements.sectionHeader }, ['Calls']),
+            initialOpenState: true
+          }, [
+            div({ style: { marginLeft: '1rem' } },
+              [makeSection('Total Call Status Counts', [
+                !_.isEmpty(calls) ? statusCell(workflow) : div({ style: { marginTop: '0.5rem' } }, ['No calls have been started by this workflow.'])
+              ]),
+              !_.isEmpty(calls) && makeSection('Call Lists', [
+                _.map(callName => {
+                  return h(Collapse, {
+                    key: callName,
+                    style: { marginLeft: '1rem', marginTop: '0.5rem' },
+                    title: div({ style: { ...Style.codeFont, ...Style.elements.sectionHeader } }, [`${callName} (× ${calls[callName].length})`]),
```

Comment:
Personal style preference, if in doubt omit parentheses
```suggestion
                    title: div({ style: { ...Style.codeFont, ...Style.elements.sectionHeader } }, [`${callName} × ${calls[callName].length}`]),
```